definitions/npm/parse-decimal-number_v1.x.x/flow_v0.25.x-/parse-decimal-number_v1.x.x.js

```diff
@@ -0,0 +1,3 @@
+declare module 'parse-decimal-number' {
+  declare module.exports: (value: string, inOptions?: any, enforceGroupSize?: boolean) => number;
```

Comment:
`inOptions` can be either `string | Array<string> | { thousands: string, decimal: string }`.

Reply:
The return value can be either `number | NaN`.

Reply:
Missing `parseDecimalNumber.factorySettings`, `parseDecimalNumber.setOptions`, `parseDecimalNumber.withOptions` signatures.